Common/tests/ListApplicationsRequestExt/ToApplicationFieldTest.cs

```diff
@@ -84,13 +84,25 @@ public static IEnumerable<TestCaseData> TestCasesInvoke()
       => new TestCaseData(field,
                           expected).SetArgDisplayNames(field.ToString());
 
-    yield return Case(ApplicationRawField.Service,
+    yield return Case(new ApplicationRawField
```

Comment:
You should keep one liner `Case` and put the `new` par the in definition of `Case`. Then `Case` would take `ApplicationRawEnumField` as input.
This should be done for every generated testcase based on this idea.